src/protagonist/API.Tests/Integration/ModifyAssetTests.cs

```diff
@@ -93,219 +63,180 @@ public class AssetTests :
+                    A<CancellationToken>._))
+            .Returns(HttpStatusCode.OK);
         
         // act
-        var response = await httpClient.AsCustomer(99).GetAsync(getUrl);
+        var content = new StringContent(hydraImageBody, Encoding.UTF8, "application/json");
+        var response = await httpClient.AsCustomer(99).PutAsync(assetId.ToApiResourcePath(), content);
         
         // assert
-        response.StatusCode.Should().Be(HttpStatusCode.NotFound);
+        response.StatusCode.Should().Be(HttpStatusCode.Created);
+        response.Headers.Location.PathAndQuery.Should().Be(assetId.ToApiResourcePath());
+        var asset = await dbContext.Images.FindAsync(assetId.ToString());
+        asset.Id.Should().Be(assetId.ToString());
+        asset.MaxUnauthorised.Should().Be(-1);
+        asset.ThumbnailPolicy.Should().Be("default");
+        asset.ImageOptimisationPolicy.Should().Be("fast-higher");
     }
 
-    
     [Fact]
-    public async Task Get_Asset_Returns_Asset()
+    public async Task Put_NewImageAsset_ReturnsEngineStatusCode_IfEngineRequestFails()
     {
-        // GET IMAGE
-        // arrange
-        var modelId = nameof(Get_Asset_Returns_Asset);
-        var id = $"99/1/{modelId}";
-        await dbContext.Images.AddTestAsset(id, customer:99, space:1);
-        await dbContext.SaveChangesAsync();
-        var getUrl = $"/customers/99/spaces/1/images/{modelId}";
+        var assetId = new AssetId(99, 1, nameof(Put_NewImageAsset_ReturnsEngineStatusCode_IfEngineRequestFails));
+        var hydraImageBody = $@"{{
+  ""@type"": ""Image"",
+  ""origin"": ""https://example.org/{assetId.Asset}.tiff""
+}}";
+        A.CallTo(() =>
+                EngineClient.SynchronousIngest(
+                    A<IngestAssetRequest>.That.Matches(r => r.Asset.Id == assetId.ToString()), false,
+                    A<CancellationToken>._))
+            .Returns(HttpStatusCode.TooManyRequests);  // Random status to verify it filters down
         
-        // act
-        var response = await httpClient.AsCustomer(99).GetAsync(getUrl);
+        var content = new StringContent(hydraImageBody, Encoding.UTF8, "application/json");
         
-        // assert
-        response.StatusCode.Should().Be(HttpStatusCode.OK);
-        var hydraImage = await response.ReadAsHydraResponseAsync<Image>();
-        hydraImage.Id.Should().EndWith(getUrl);
+        // Act
+        var response = await httpClient.AsCustomer(99).PutAsync(assetId.ToApiResourcePath(), content);
+        
+        response.StatusCode.Should().Be(HttpStatusCode.TooManyRequests);
+        response.Headers.Location.Should().BeNull();
+        var asset = await dbContext.Images.FindAsync(assetId.ToString());
+        asset.Id.Should().Be(assetId.ToString());
     }
     
-    [Fact]
-    public async Task Get_Assets_In_Space_Returns_NotFound_For_Missing_Space()
+    [Fact(Skip = "Is this an expected behaviour?")]
```

Comment:
@tomcrane - should we set the `Asset.Error` property if a) The call to enqueue or ingest fails AND b) the `Asset.Error` property is empty? 

I don't think deliverator does this now and always assumes that the engine request will succeed, however I think it could be useful. Without it an image could sit in a "ingesting" limbo